### PR TITLE
Arrhythmic Knife

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
It randomly slows and speeds you up. Made it for @TechnoAlchemisto (his sprites). 

Can't put it away for BALANCE (waiting out the slows)

If this is too silly for a chaplain weapon I'll just make it admin only.

Please send me my money now Techno thank you

### CODE REVIEW

**Comment by PKPenguin321** [comment]
test

**Comment by KorPhaeron** [comment]
Yeah that thanks

**Comment by GunHog** [comment]
And we have hit the PNR! Chaplain is now a **Holy warrior**, and there is no going back now!

**Comment by KorPhaeron** [comment]
What is a PNR

**Comment by RemieRichards** [comment]
@KorPhaeron point of no return.
it took me a second or too aswell.

File: code/game/objects/items/weapons/holy_weapons.dm [hunk 1]
```diff
@@ -262,4 +262,28 @@
...
 	icon_state = "bostaff0"
+	item_state = "bostaff0"
 
+/obj/item/weapon/nullrod/tribal_knife
+	icon_state = "crysknife"
+	item_state = "crysknife"
+	name = "arrhythmic knife"
+	w_class = 5
+	desc = "They say fear is the true mind killer, but stabbing them in the head works too. Honour compels you to not sheathe it once drawn."
+	slot_flags = SLOT_BELT
```
Comment (KazeEspada) [inline code/game/objects/items/weapons/holy_weapons.dm]: Fits on belt tho?

Comment (tkdrg) [inline code/game/objects/items/weapons/holy_weapons.dm]: also can be dropped
but that might be intended

**Comment by tkdrg** [comment]
also conflicts

**Comment by KorPhaeron** [comment]
@tkdrg 

I'll fix it after chair inhands get merged since it will just conflict again otherwise

**Comment by tkdrg** [comment]
if you wait until nobody has inhand PRs open it will never get merged but ok

File: tgstation.dme [hunk 2]
```diff
@@ -986,7 +986,6 @@
 #include "code\modules\events\meateor_wave.dm"
 #include "code\modules\events\meteor_wave.dm"
 #include "code\modules\events\operative.dm"
-#include "code\modules\events\portal_storm.dm"
```
Comment (KorPhaeron) [inline tgstation.dme]: Oh godamnit

**Pull request merged**